test/unit/presenters/publishing_api/consultation_presenter_test.rb

```diff
@@ -180,6 +180,26 @@ class BasicConsultationTest < TestCase
     test "validity" do
       assert_valid_against_schema presented_content, "consultation"
     end
+
+    test "it presents the base_path if locale is :en" do
+      assert_equal "/government/consultations/consultation-title", presented_content[:base_path]
+    end
+
+    test "it presents the base_path with locale if non-english" do
+      with_locale("it") do
+        assert_equal "/government/consultations/consultation-title.it", presented_content[:base_path]
+      end
+    end
+
+    test "it presents the default global process wide locale as the locale of the consultation" do
+      assert_equal "en", presented_content[:locale]
+    end
+
+    test "it presents the selected global process wide locale as the locale of the consultation" do
+      with_locale("it") do
+        assert_equal "it", presented_content[:locale]
+      end
+    end
```

Comment:
This set of tests would probably read better as two contexts:

```ruby
context "with default locale" do
  test "it presents the default global process wide locale as the locale of the consultation" do
  # 
  # etc
end

context "with overridden locale" do
  with_locale("it") do
    test "it presents ..."
    # etc
```

Reply:
Unfortunately it seems that we do not have easy access to `context` blocks in this test file without introducing a `describe` block also, and the objects the tests assert against are method calls in the test's parent class (`BasicConsultationTest < TestCase`) which we then lose access to. 

The setup and running of these tests is relatively complex!

Reply:
Oh, I had no idea! Fair enough :D